src/components/database/add-collection-form/AddCollectionForm.jsx

```diff
@@ -98,44 +110,46 @@ const AddCollectionForm = ({ form, editMode, projectId, selectedDB, handleSubmit
               </span>
             )}
           </Form.Item>
-          <Row>
-            <Col span={12}>
-              <FormItemLabel name="Schema" />
-              <CodeMirror
-                value={schema}
-                options={{
-                  mode: { name: "javascript", json: true },
-                  lineNumbers: true,
-                  styleActiveLine: true,
-                  matchBrackets: true,
-                  autoCloseBrackets: true,
-                  tabSize: 2,
-                  autofocus: true
-                }}
-                onBeforeChange={(editor, data, value) => {
-                  setSchema(value)
-                }}
-              />
-            </Col>
-            <Col>
-              <FormItemLabel name="Rule" />
-              <CodeMirror
-                value={rule}
-                options={{
-                  mode: { name: "javascript", json: true },
-                  lineNumbers: true,
-                  styleActiveLine: true,
-                  matchBrackets: true,
-                  autoCloseBrackets: true,
-                  tabSize: 2,
-                  autofocus: false
-                }}
-                onBeforeChange={(editor, data, value) => {
-                  setRule(value)
-                }}
-              />
-            </Col>
-          </Row>
+          <FormItemLabel name="Schema" />
+          <CodeMirror
+            value={schema}
+            options={{
+              mode: { name: "javascript", json: true },
+              lineNumbers: true,
+              styleActiveLine: true,
+              matchBrackets: true,
+              autoCloseBrackets: true,
+              tabSize: 2,
+              autofocus: true
+            }}
+            onBeforeChange={(editor, data, value) => {
+              setSchema(value)
+            }}
+          />
+          <Checkbox
+            checked={checked}
+            onChange={e =>
+              getChecked(!checked)
+            }
+          >Apply default security rules</Checkbox>
```

Comment:
UI: give some spaces above and below checkbox

when add table  with custom rules and click on edit it is by default checked it should not be checked by default when the custom rules are defined

when creating table with default rules checked and click on edit and uncheck apply default rules it shows empty